### DISCUSSION TITLE
Unable to sign-in after registration

### DISCUSSION THREAD

**Discussion by beddows** [AntlerVC/firetable#1157] [Support / Q&A] [answered]
Hello, I just registered as a new user using Github. Everything went smoothly with the onboarding process. At the end, it requested I sign in again and when I do, I get the attached blank screen. I've deleted cookies for rowy and github, but it doesn't help.
![Xnip2022-12-21_07-20-34](https://user-images.githubusercontent.com/13971636/208895131-4087626b-cd6d-4875-8f7b-a6700c2755ce.jpg)

**Comment by beddows** [comment]
Not sure why that screenshot is so large... Anyways, I checked Github Integrations-Applications and Rowy is there.

**Comment by beddows** [comment]
Just reading #837 - I have not previously signed in with Google or Microsoft.

**Comment by shamsmosowi** [comment, marked as answer]
Hi @beddows are you able to log in using your google account?

**Comment by sebamondi** [comment]
Hi @shamsmosowi 
I have the same issue with mi google account. 
I was using the rowy app in nov-22 without problem, but now I can not login...
How can I solve this login issue?

Thank you.

Best regard

**Comment by BeeBombshell** [comment]
Hey @beddows, @sebamondi, is this still an issue?

**Reply by beddows** [reply]
@BeeBombshell I haven't tried to sign in, I'm using a different tech stack now.
